### DISCUSSION TITLE
`Last Name` asked twice in `create-admin`

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [neonexus/sails-react-bootstrap-webpack#136] [Q&A]
Hi @neonexus

For some reason, when I call `npm run create:admin` to create my first admin user, `lastName` is asked twice. Not sure why?
Do you have any idea?

Thanks

**Comment by neonexus** [comment]
@MaheshkumarSundaram I'll look into this, but can you tell me if there were any special characters in the names given?

**Reply by MaheshkumarSundaram** [reply]
Thank you. I didn't give any special character at all. Just alphabets.

I am using Windows OS (Just letting you know, although I highly doubt any impact because of this)

**Comment by neonexus** [comment]
I can't seem to find the error. What version was this on?

**Reply by neonexus** [reply]
@MaheshkumarSundaram Can you give reproduction steps / are you able to repeat the issue?

**Reply by MaheshkumarSundaram** [reply]
@neonexus,

Its not creating any issues per se. I use Windows.
In `cmd`, when I use the command `npm run create:admin`, as specified it asks for all the necessary details one by one.
`Last Name` is the final input question. 

- I provide the value for `Last Name` and press Enter. 
- Now it looks like nothing is being done (no error either). 
- So I press Enter again, it prompts for the `Last Name` input again. 
- Once again, I provide the value and press Enter.#
- Now, the user gets created with confirmation.

There is no error but for some reason, `Last Name` asked twice. Tbh I haven't debugged this myself as its not creating any issue. Sorry.

**Reply by neonexus** [reply]
@MaheshkumarSundaram AH! This is a Win vs. the world issue. This may be possible to correct, but also time consuming. If everything still works; then for now, I'm going to leave it. I have a feeling it's likely something simple like a missing character the Win cmd isn't receiving.

**Reply by MaheshkumarSundaram** [reply]
@neonexus Yeah. Its so minor. Sometime I'll debug it and let you know if I figure out why. Thank you
